nat-lab/tests/test_notification_center.py

```diff
@@ -24,6 +23,14 @@ class MachineResponse:
     traffic_routing_supported: bool
 
 
+bearer_authorization_header = "Bearer StellarPass"
```

Comment:
Maybe `Bearer token:StellarPass`?

Reply:
Same comment and discussion here:
https://github.com/NordSecurity/libtelio/pull/908#discussion_r1840489068